lib/activity.js

```diff
@@ -20,6 +24,11 @@ export class ActivityClient extends Client {
             .then(res => res.data)
             .then(data => this.afterDataProcessed(endpoint, data));
     }
+
+    getLiveFeed(opts = {}) {
+        const endpoint = this.getEndpoint('getLiveFeed', opts);
+        return this.fetch(endpoint);
```

Comment:
Think you also need to return `res.data` like in L24 in the `getNotifications` and maybe run it through the `afterDataProcessed` function as well (L25), but that might be optional.

Reply:
I'm using it locally and I haven't had to return `res.data` - the endpoint returns an array - https://worldapi.kano.me/docs/#/activity/ActivityController_getActivity